pkg/jobrunaggregator/jobrunbigqueryloader/types.go

```diff
@@ -105,11 +110,20 @@ func (v *testRunRow) Save() (map[string]bigquery.Value, string, error) {
 		status = "Passed"
 	}
 
+	var testSuite_str string
+	if len(v.testSuites) == 1 {
+		testSuite_str = v.testSuites[0]
+	} else {
+		// There is generally a single test suite.  But if there are more, we will
+		// concat them to make a unique test suite name.
+		testSuite_str = strings.Join(v.testSuites, "|")
```

Comment:
I'm ok starting with this. I'm also ok choosing more than one character as the limiter (I would probably choose at least two characters, like `|||`).